client/my-sites/email/titan-mail-quantity-selection/index.jsx

```diff
@@ -243,151 +188,149 @@ class TitanMailQuantitySelection extends React.Component {
-				<span>
+					) }{ ' ' }
+					{ purchaseCost.amount < renewalCost.amount
+						? translate(
+								'This is less than the regular price because you are only charged for the remainder of the current month.'
+						  )
+						: translate(
+								'This is more than the regular price because you are charged for the remainder of the current month plus any additional month until renewal.'
+						  ) }{ ' ' }
 					{ translate(
-						'All of your mailboxes are due to renew at {{strong}}%(price)s/mailbox/month{{/strong}} on %(date)s',
+						'All of your mailboxes are due to renew at the regular price of {{b}}%(fullPrice)s{{/b}} per mailbox when your subscription renews on {{b}}%(expiryDate)s{{/b}}.',
 						{
 							args: {
-								date: moment( expiryDate ).format( 'LL' ),
-								price: renewalCost.text,
+								proratedPrice: purchaseCost.text,
+								fullPrice: renewalCost.text,
+								expiryDate: moment( expiryDate ).format( 'LL' ),
 							},
 							components: {
-								strong: <strong />,
+								b: <strong />,
 							},
-							comment:
-								'%(price)s is a formatted price for each mailbox, e.g. $3.50, €3.75, or PLN 4.50; ' +
-								'%(date)s is a localized date of the rough form: 17 February 2021',
 						}
 					) }
-				</span>
-				<CompactCard className="titan-mail-quantity-selection__prorating-details">
-					<Gridicon icon="info-outline" size={ 18 } />
-					<span>{ prorationMessage }</span>
-				</CompactCard>
-			</React.Fragment>
+				</>
+			</Notice>
 		);
 	}
 
 	renderForm() {
-		const { isLoadingDomains, selectedDomainName, titanMonthlyProduct, translate } = this.props;
+		const { isLoadingDomains, titanMonthlyProduct, translate } = this.props;
 
 		if ( isLoadingDomains || ! titanMonthlyProduct ) {
 			return <AddEmailAddressesCardPlaceholder />;
 		}
 
 		return (
 			<>
-				<SectionHeader label={ translate( 'Choose Mailbox Quantity' ) } />
+				<SectionHeader label={ translate( 'Add New Mailboxes' ) } />
 
 				<Card>
-					<div className="titan-mail-quantity-selection__domain-info">
-						{ translate( "You're adding mailboxes for {{strong}}%(domainName)s{{/strong}}", {
-							args: {
-								domainName: selectedDomainName,
-							},
-							components: {
-								strong: <strong />,
-							},
-							comment: '%(domainName)s is a domain name',
-						} ) }
-					</div>
-					<div className="titan-mail-quantity-selection__mailbox-info">
-						{ this.renderCurrentMailboxCounts() }
-					</div>
-					<div>
-						<FormLabel>{ translate( 'Number of new mailboxes to add' ) }</FormLabel>
+					<div className="titan-mail-quantity-selection__form">
+						<FormLabel htmlFor="quantity">{ translate( 'Number Of Mailboxes' ) }</FormLabel>
+
 						<FormTextInput
 							name="quantity"
+							id="quantity"
 							type="number"
 							min="1"
 							max="50"
 							step="1"
-							placeholder={ translate( 'Number of new mailboxes' ) }
+							placeholder={ translate( 'Number of mailboxes' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.